### DISCUSSION TITLE
好像本地自己安装一个应用并配置反向代理导致了1panel安装的应用都没法使用一键部署创建网站了

### DISCUSSION THREAD

**Discussion by IshuaWang** [1Panel-dev/1Panel#6636] [General]
不知道是我自己的问题还是确实是一个bug。就是在创建网站的时候，选择应用只有我本地自己安装的应用可以选，1panel安装的那些好像都看不到了。

**Comment by IshuaWang** [comment]
好像是有的应用不行，有的可以，比如alist， Material for MkDocs就不行，有的就行

**Reply by wanghe-fit2cloud** [reply]
应用商店安装应用时，勾选 `端口外部访问` 选项，如图：
<img width="485" alt="image" src="https://github.com/user-attachments/assets/fd7b2db5-e3fc-4e69-b7d8-4a276b062644">

**Reply by IshuaWang** [reply]
应该不是这个问题，我记得我安装的时候是勾选了这个选项的，而且我发现防火墙显示的打开窗口和腾讯云控制台显示的不一样。

**Comment by DaisenTravel** [comment]
这是来自QQ邮箱的假期自动回复邮件。
 
您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。
